content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+This means that anyone who can update (or patch or create) a namespace can also modify the Pod
+Security level for that namespace, which could be used to bypass a more restrictive policy. Before
+proceeding, ensure that only trusted, privileged users have these namespace permissions. It is not
+recommended to grant these powerful permissions to users that shouldn't have elevated permissions,
+but if you must you will need to use an [admission webhook][dynamic-admission] to place additional
+restrictions on Pod Security labels.
+
+## 2. Simplify & Standardize PodSecurityPolicies {#simplify-psps}
+
+In this section, you will reduce mutating PodSecurityPolicies and remove options that are outside
+the scope of the PodSecurityStandards. You should make the changes recommended here to an offline
+copy of the original PodSecurityPolicy being modified. The cloned profile should have a different
+name that is alphabetically before the original (for example, prepend a `0` to it). Do not create the
+new policies in Kubernetes yet - that will be covered in the [Rollout the updated
+policies](#psp-update-rollout) section below.
+
+### 2.a. Eliminate purely mutating fields
+
+If a PodSecurityPolicy is mutating pods, then you could end up with pods that don't meet the Pod
+Security level requirements when you finally turn PodSecurityPolicy off. In order to avoid this, you
+should eliminate all PSP mutation prior to switching over. Unfortunately PSP does not cleanly
+separate mutating & validating fields, so this is not a straightforward migration.
+
+You can start by eliminating the fields that are purely mutating, and don't have any bearing on the
+validating policy. These fields (also listed in the [table below](#psp-fields)) are:
+
+- `.spec.defaultAllowPrivilegeEscalation`
+- `.spec.runtimeClass.defaultRuntimeClassName`
+- `.metadata.annotations['seccomp.security.alpha.kubernetes.io/defaultProfileName']`
+- `.metadata.annotations['apparmor.security.beta.kubernetes.io/defaultProfileName']`
+- `.spec.defaultAddCapabilities` - Although technically a mutating & validating field, these should
+  be merged into `.spec.allowedCapabilities` which performs the same validation without mutation.
+
+{{< caution >}}
+Removing these could result in workloads missing required configuration, and cause problems. See
+[Rollout the updated policies](#psp-update-rollout) below for advice on how to roll these changes
+out safely.
+{{< /caution >}}
+
+### 2.b. Eliminate options not covered by the Pod Security Standards
+
+There are several fields in PodSecurityPolicy that are not covered by the Pod Security Standards. If
+you must enforce these options, you will need to supplement Pod Security Admission with an
+[admission webhook][dynamic-admission], which is outside the scope of this guide.
+
+First, you can remove the purely validating fields that Pod Security does not cover. These fields
```

Comment:
What does "Pod Security" stand for here? The "Pod Security Admission" or "PodSecurityPolicy"?